blockchain/vm/interpreter.go

```diff
@@ -44,8 +44,8 @@ type Config struct {
 	// RunningEVM is to indicate the running EVM and used to stop the EVM.
 	RunningEVM chan *EVM
 
-	// UseOpcodeComputationCost is to enable applying the opcode computation cost limit.
-	UseOpcodeComputationCost bool
+	// ComputationCostLimit is the limit of the total computation cost of a transaction. Set to Zero of Infinite to disable the computation cost limit.
```

Comment:
```suggestion
	// ComputationCostLimit is the limit of the total computation cost of a transaction. Set to Zero or Infinite to disable the computation cost limit.
```

Reply:
revised it at https://github.com/klaytn/klaytn/pull/2037/commits/8d9deb0ecb40ec89244fd97a5b2c3800560685fe